EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloLayer1Collections.cc

```diff
@@ -0,0 +1,14 @@
+#include "FWCore/Framework/interface/Event.h"
+
+#include "CaloLayer1Collections.h"
+
+namespace l1t {
+   namespace stage2 {
+      CaloLayer1Collections::~CaloLayer1Collections()
+      {
+         event_.put(std::move(ecalDigis_));
```

Comment:
hi @rekovic - was looking at this again - is this construct of putting stuff in the event as the variable goes out of scope something already in place elsewhere? its very confusing (and relies on a certain usage pattern of the class)

Reply:
hi @davidlange6 Yes. This is how the unpackers are putting data in Event, like we have for example for [BMTF](https://github.com/cms-sw/cmssw/blob/CMSSW_9_4_X/EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFCollections.cc), [EMTF](https://github.com/cms-sw/cmssw/blob/CMSSW_9_4_X/EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFCollections.cc).